pwndbg/gdblib/config.py

```diff
@@ -82,7 +64,7 @@ def native_default(self):
             self.param.default, param_class=self.param.param_class
         )
 
-    def __get_set_string_gdb_gte_9(self) -> str:
+    def get_set_string(self) -> str:
         """Handles the GDB `set <param>` command for GDB >= 9"""
```

Comment:
This comment no longer needs to reference GDB 9, it can just be "Handles the GDB `set <param>`"

Reply:
Updated. Thanks